lib/site_settings/type_supervisor.rb

```diff
@@ -37,7 +37,6 @@ def self.types
       color: 22,
       simple_list: 23,
       emoji_list: 24,
-      html: 25
```

Comment:
As it is with all enum types - I'm going to ask: do we want to re-use that number in the future or do we want to leave a comment/rename that to e.g. `html_deprecated`?